src/main/resources/templates/player/assignment/sequence/phase/evaluation/method/draxo/_draxo-show.html

```diff
@@ -15,65 +15,63 @@
                             <i class="red x icon" th:if="${optionType?.name == 'NEGATIVE'}"></i>
                             <i class="question icon" th:if="${optionType?.name == 'UNKNOWN'}"></i>
-
-                            <div class="content">
-                                <div class="title" th:text="${criteria}">R</div>
-                                <div class="description"
-                                     th:text="#{${criteria.getMessageI18nKey(T(org.elaastic.questions.assignment.sequence.peergrading.draxo.criteria.CriteriaMessageKey).Header)}}">
-                                    ...
-                                </div>
-                            </div>
                         </div>
-                    </th:block>
-                </div>
+                    </div>
+                </th:block>
+            </div>
 
-                <th:block th:with="rejectedCriteria = ${model.draxoEvaluation.getRejectedCriteria()}">
-                    <div class="ui form" th:if="${rejectedCriteria}">
-                        <div class="inline field">
-                            <label th:text="#{${rejectedCriteria.getMessageI18nKey(T(org.elaastic.questions.assignment.sequence.peergrading.draxo.criteria.CriteriaMessageKey).Question)}}+' :'"></label>
-                            <span th:text="#{${model.draxoEvaluation.get(rejectedCriteria).codeI18n}}"></span>
+            <th:block th:with="rejectedCriteria = ${model.draxoEvaluation.getRejectedCriteria()}">
+                <div class="comment-area" th:if="${rejectedCriteria}">
+                    <div class="comment-header">
+                        <h4 th:text="#{${rejectedCriteria.getMessageI18nKey(T(org.elaastic.questions.assignment.sequence.peergrading.draxo.criteria.CriteriaMessageKey).Question)}}+' :'">
+                            The answer is readable :</h4>
+                        <div class="ui label" th:text="#{${model.draxoEvaluation.get(rejectedCriteria).codeI18n}}">
+                            No
                         </div>
-
-                        <div class="field" th:if="${model.draxoEvaluation.getExplanation()}">
-                            <label th:text="#{draxo.feedback.label}">Feedback:</label>
-                            <p th:text="${model.draxoEvaluation.getExplanation()}"></p>
+                    </div>
+                    <div class="comment" th:if="${model.draxoEvaluation.getExplanation()}">
+                        <h4 th:text="#{draxo.feedback.label}">Feedback:</h4>
+                        <div class="ui label" th:text="${model.draxoEvaluation.getExplanation()}">
+                            I am a relevant and constructive comment, in order to explain why the answer provided above is not decipherable.
+                            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
```

Comment:
Maybe remove Lorem ipsum ? (You can leave a default message (_I am a relevant and constructive comment, in order to explain why the answer provided above is not decipherable._)  in case `th:text` is not working)